pages/filter.tsx

```diff
@@ -101,12 +131,15 @@ export const FilterPage: NextPage = () => {
 
   return (
     <Layout>
+      <Heading>
+        <h1>Filter</h1>
+      </Heading>
       <Top>
-        <h2>Filter</h2>
+        <Link href={'/'}>{'< back'}</Link>
```

Comment:
Back? Also you can use Arrow.tsx for the icon, otherwise a SR will say "link less back"